tests/test_featureprofiletools_update_profiles_inventories.py

```diff
@@ -0,0 +1,57 @@
+from langworld_db_data.featureprofiletools.update_profiles_inventories import (
+    rename_value_in_profiles_and_inventories,
+)
+from tests.paths import DIR_WITH_FEATURE_PROFILE_TOOLS_TEST_FILES
+from tests.test_helpers import check_existence_of_output_csv_file_and_compare_with_gold_standard
+
+DIR_WITH_TEST_UPDATE_PROFILES_INVENTORIES = (
+    DIR_WITH_FEATURE_PROFILE_TOOLS_TEST_FILES / "update_profiles_inventories"
+)
+
```

Comment:
I think you can remove blank lines between declarations of constants

Reply:
Done, and I also removed an outdated commentary from the module